### PR TITLE
Use right stick for aim, defaults to left stick (move)

### PR DESCRIPTION
**Pull request by luoyang9**

### CODE REVIEW

File: Assets/Scripts/InputActionMapper.cs [hunk 1]
```diff
@@ -84,7 +84,7 @@ private void Awake()
                     break;
                 default:
                     var stickDirection = aim.ReadValue<Vector2>();
-                    direction = stickDirection == Vector2.zero ? new Vector2(0, 1) : stickDirection;
+                    direction = stickDirection == move.ReadValue<Vector2>() ? new Vector2(0, 1) : stickDirection;
```
Comment (mrdofu) [inline Assets/Scripts/InputActionMapper.cs]: shouldn't you replace the default `Vector2(0, 1)`?

Comment (luoyang9) [reply Assets/Scripts/InputActionMapper.cs]: oops

File: Assets/Scripts/InputActionMapper.cs [hunk 2]
```diff
@@ -84,7 +84,7 @@ private void Awake()
                     break;
                 default:
                     var stickDirection = aim.ReadValue<Vector2>();
-                    direction = stickDirection == Vector2.zero ? new Vector2(0, 1) : stickDirection;
+                    direction = stickDirection == Vector2.zero ? move.ReadValue<Vector2>() : stickDirection;
```
Comment (mrdofu) [inline Assets/Scripts/InputActionMapper.cs]: what happens when both sticks are zero?

Comment (luoyang9) [reply Assets/Scripts/InputActionMapper.cs]: now defaults to most recent direction

**Pull request merged**